monai/_version.py

```diff
@@ -0,0 +1,520 @@
+
```

Comment:
Should we add MONAI copyright in this file?

Reply:
i don't think so, this file is in public domain we can't claim copyright